Tests/test_PDB_MMCIF2Dict.py

```diff
@@ -21,6 +21,9 @@
 
 from Bio.PDB.MMCIF2Dict import MMCIF2Dict
 
+import io
```

Comment:
Using ``io.StringIO`` makes Python 2 and 3 at once tricky, which would explain why you needed the explicit unicode strings later.

Why we normally use ``from Bio._py3k import StringIO`` which does ``from StringIO import StringIO`` under Python 2 or ``from io import StringIO`` under Python 3.

Either was seems fine (and the difference only matters under Python 2 anyway which will eventually go away).

Reply:
Ah-ha! I thought there might be something like that available, but missed `Bio._py3k`. I don't develop on Python 2, but the build bot did its job nicely and let me know there was a problem. I don't mind switching to `Bio._py3k` if you consider that more idiomatic for biopython.

Reply:
Sure - my guess is you can use ``from Bio._py3k import StringIO`` and remove the ``u`` prefixes.

Note that in general I would recommend using the third-party library ``six`` for this kind of compatibility shim, but we wanted to keep the Biopython dependencies to a minimum and have a few other more project specific hacks under ``Bio._py3k`` anyway.